apps/mobile/src/libs/components/screen-wrapper/screen-wrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+
+import { View } from "~/libs/components/components";
+import { ScreenWrapperEdge } from "~/libs/enums/app/screen-wrapper-edge.enum";
+import { ValueOf } from "~/libs/types/types";
+
+const ZERO_PADDING = 0;
+
+type Properties = {
+	children: React.ReactNode;
+	edges?: ValueOf<typeof ScreenWrapperEdge>[];
```

Comment:
You can use `Edge` type from `react-native-safe-area-context` so the ScreenWrapperEdge is redundant.

Also, it would be great to add the default value like:
`const DEFAULT_SAFE_AREA_EDGES: Edge[] = ['top', 'bottom', 'left', 'right'];`

Reply:
change to Edge type, Add default value